source/includes/v2/_catalog.md

```diff
@@ -104,167 +108,170 @@ curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets?l
-
-<aside>
-It is not possible to retrieve all datasets from a domain with this API. To do that, export endpoints must be used.
-</aside>
-
-##### HTTP Request
-`GET /api/v2/catalog/datasets`
-
-##### URL Parameters
-
-Parameter | Default | Description
---------- | ------- | -----------
-`where` | None | Filter expression used to restrict returned datasets ([ODSQL documentation](#where-clause))
-`refine` | None | Refine the result set on a given facet value (see [refine in Facet documentation](#refine))
-`exclude` | None | Exclude a given facet value from the result set (see [exclude in Facet documentation](#exclude))
-`offset` | 0 | Index of the first item to return
-`limit` | 10 | Number of items to return. Max value: 100
-`include_app_metas` | false | Explicitly request application metadata for each datasets
-`timezone` | UTC | Timezone applied on datetime fields in query and response
-
-<aside>
-The value of both `offset` and `limit` parameters must not exceed 10000. Use the export API to download all datasets.
-</aside>
-
-## Aggregating datasets
-
-> Aggregation query without group_by
+> Grouping results (here, by `feature` metadata) with a single group_by
 
 ```shell
-curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/aggregates/?select=count(*) as count'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/?select=features,count(*) as count&group_by=features'
 ```
 
 > API Response
 
 ```json
 {
-    "aggregations": [
-        {
-            "count": 12
+    "datasets": [
+        {"dataset":
+            {
+                "count": 12,
+                "features": "analyze"
+            }
+        },
+        {"dataset":
+            {
+                "count": 4,
+                "features": "geo"
+            },
+        },
+        {"dataset":
+            {
+                "count": 9,
+                "features": "timeserie"
+            }
         }
     ]
 }
 ```
 
-> Aggregation query with a single group_by
+> Performing an analytic on grouped results (here, a catalog datasets `count`), without grouping by a dedicated metadata
 
 ```shell
-curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/aggregates/?select=features,count(*) as count&group_by=features'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/?select=count(*) as count&group_by'
 ```
 
 > API Response
 
 ```json
 {
-    "aggregations": [
-        {
-            "count(*)": 12,
-            "features": "analyze"
-        },
-        {
-            "count(*)": 4,
-            "features": "geo"
-        },
-        {
-            "count(*)": 9,
-            "features": "timeserie"
+    "datasets": [
+        {"dataset":
+            {
+                "count": 12
+            }
         }
     ]
 }
 ```
 
-
-> Invalid aggregation with a selected field not present in group_by
+> Invalid analytic on grouped results because no grouping is asked
```

Comment:
I would add an 's' to analytic, because we need the noun here, not the adjective.

Reply:
Analytic function?
We can discuss this further to make this consistent throughout the docs.